dropwizard-oauth/src/main/java/smartthings/dw/oauth/SpringSecurityAuthenticator.java

```diff
@@ -50,8 +47,9 @@ public SpringSecurityAuthenticator(AuthConfiguration config, AsyncHttpClient cli
 				.setRequestTimeout(timeout)
 				.setRealm(realm)
 				.addHeader("Accept", "application/json")
+                .addHeader("Content-Type", "application/x-www-form-urlencoded")
 				.addHeader(LoggingContext.CORRELATION_ID_HEADER, LoggingContext.getLoggingId())
-				.addFormParam("token", URL_ESCAPER.escape(token))
+				.addFormParam("token", token)
```

Comment:
What's driving the removal of the url escaping?

Reply:
Assumed role tokens appear to be double escaped causing auth checks to fail on them

Reply:
To answer my question - looks like many years ago this method used to be a GET operation with the token in URL.  The escaping was needed due to how URL construction used to work.  Along the way it was changed to a form post, and the url escaping was copy/pasted forward.  

👍 on fixing.